ruma-federation-api/src/membership/create_leave_event.rs

```diff
@@ -0,0 +1,4 @@
+//! Endpoint to submits a signed leave event to the resident server for it to accept it into the room's graph.
```

Comment:
Can we not use the "resident" terminology the spec uses? I find "receiving" to be the much clearer term.